CONTRIBUTING.md

```diff
@@ -1,19 +1,65 @@
-Code formatting
-===============
+# How to contribute
 
-Go provides a tool, `go fmt`, which facilitates a standardised format to Go 
-source code. SQLair has two additional policies:
+Thanks for your interest in SQLair! Contributions like yours make good projects
+great.
 
-Imports
--------
+When contributing, you must abide by the [Ubuntu Code of
+Conduct](https://ubuntu.com/community/ethos/code-of-conduct).
 
```

Comment:
Put this in its own separate Code of conduct section?

Reply:
Added title above.